plugins/woocommerce-blocks/package.json

```diff
@@ -307,7 +307,7 @@
 		"wordpress-components-slotfill": "npm:@wordpress/components@wp-6.5"
```

Comment:
It seemed like there's a specific version required originating from [this PR](https://github.com/woocommerce/woocommerce/pull/47105) so I didn't want to touch it - it would change the build